packages/auth-express/readme.md

```diff
@@ -0,0 +1,397 @@
+// Creates the following routes:
+// - GET /auth/signin: Redirects to built-in UI's sign in page
+// - GET /auth/signup: Redirects to built-in UI's sign up page
+// - GET /auth/callback: Handles successful authentication. Typically you'll create a new user if `isSignUp` is true, and redirect appropriately.
+```
+
+### Custom UI: Email and password `createEmailPasswordRouter`
+
+- `routerPath: string`, required, This is the path relative to the `baseUrl` given that was configured when creating the `ExpressAuth` object that this router will be attached to. This is used to build the URL to the email verification path configured by the router factory.
+- `signIn: (express.RouteHandler | express.ErrorHandler)[]`, required, Attached middleware executes when sign-in attempt succeeds. Typically you will redirect the user to your application here.
+- `signUp: (express.RouteHandler | express.ErrorHandler)[]`, required, Attached middleware executes when sign-up attempt succeeds. Typically you will redirect the user to your application here.
+- `verify: (express.RouteHandler | express.ErrorHandler)[]`, Attached middleware executes after the user verifies their email successfully. Typically you will redirect the user to your application here.
+- `sendPasswordResetEmail: (express.RouteHandler | express.ErrorHandler)[]`, Attached middleware executes after the user sends a password reset. Typically you will redirect the user to some success page containing instructions to look for a reset email.
+- `resetPassword: (express.RouteHandler | express.ErrorHandler)[]`, Attached middleware executes after the user successfully resets their password. Typically you will redirect the user to your application here.
+- `resendVerificationEmail: (express.RouteHandler | express.ErrorHandler)[]`, Attached middleware executes after the user resends their email verification email. Typically you will redirect the user to some success page, or if you do not _require_ verification to use your app, you can redirect to your app here.
+
+
+```ts
+const emailPasswordRouter = auth.createEmailPasswordRouter("/auth/email-password", {
+  signIn: [
+    (req: expressAuth.AuthRequest, res) => {
+      res.redirect("/");
+    },
+  ],
+  signUp: [
+    (req: expressAuth.AuthRequest, res) => {
+      res.redirect("/onboarding");
+    },
+  ],
+  verify: [
+    (req: expressAuth.AuthRequest, res) => {
+      res.redirect("/");
+    },
+  ],
+  sendPasswordResetEmail: [
+    (req: expressAuth.AuthRequest, res) => {
+      res.redirect("/email-success");
+    },
+  ],
+  resetPassword: [
+    (req: expressAuth.AuthRequest, res) => {
+      res.redirect("/email-success");
+    },
+  ],
+  resendVerificationEmail: [
+    (req: expressAuth.AuthRequest, res) => {
+      res.redirect("/");
+    },
+  ],
+});
+
+app.use(emailPasswordRouter);
+// This creates the following routes:
+// - POST /auth/email-password/signin
+// - POST /auth/email-password/signup
+// - POST /auth/email-password/verify
```

Comment:
```suggestion
// - GET /auth/email-password/verify
```
This should be a get request right? since it's the link that gets sent in the email.